packages/jsx/src/target/html/components/tabular.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { PretextStateContext } from "../state";
+import { ReplacerComponent } from "../replacers/replacer-factory";
+import { XastElement } from "@pretext-book/jsx/src/utils/xast";
+
+export const Cell: ReplacerComponent = function ({ node }) {
+    const state = React.useContext(PretextStateContext);
+    return (
+        <td className="b0 l l0 lines m r0 t0">
+            {state.processContent(node.children)}
+        </td>
+    );
+};
+
+export const CellHeader: ReplacerComponent = function ({ node }) {
+    const state = React.useContext(PretextStateContext);
+    return (
+        <th scope="col" className="b0 l l0 lines m r0 t0">
+            {state.processContent(node.children)}
+        </th>
+    );
+};
+
+export const Row: ReplacerComponent = function ({ node }) {
+    const isHeader = node.attributes?.header == "yes"
+    const cells = node.children.filter((n) :n is XastElement=>n.type == "element" && n.name=="cell")
+    if (isHeader) {
+        return (
+            <tr className="header-horizontal">
+                {cells.map(c=>CellHeader({node: c}))}
```

Comment:
This should be
```
c => <CellHeader node={c} />
```
Since it's in an array, React actually wants a `key` attribute as well, so I wold probably do
```
(c, i) => <CellHeader node={c} key={i} />
```

Reply:
Thanks. This is moot, unless I revert the change I just made to using a single component (since tabular state is needed to configure `td`s/`th`s).